packages/material-ui-lab/src/SliderUnstyled/ValueLabelUnstyled.js

```diff
@@ -39,8 +40,9 @@ function ValueLabel(props) {
       className: clsx(children.props.className),
     },
     <Root className={clsx(classes.offset, className)} theme={theme}>
-      <span className={classes.circle}>
+      <span className={classes.container}>
```

Comment:
It seems like we removed and added classes but I see no documentation changed. Is this not something that is auto-generated or available in types? 

/cc @mnajdova

Reply:
These classes are not the props classes:
```
const classes = useValueLabelClasses(props);
```

Reply:
So there are no classnames that can target the `ValueLabel` componnet?